test/backend_binary_elementwise.in.cpp

```diff
@@ -67,36 +67,6 @@ NGRAPH_TEST(${BACKEND_NAME}, add)
                                   (test::NDArray<float, 2>({{6, 8}, {10, 12}})).get_vector()));
 }
 
-NGRAPH_TEST(${BACKEND_NAME}, dot_add)
```

Comment:
This test should not be removed. It is still used. I would like to be able to run all dot* unit tests by typing BACKEND.dot*

Reply:
It was added by us to trigger MLIR compilation for a function, that's why I moved it to `backend_mlir.in.cpp`

Reply:
👍